### DISCUSSION TITLE
Fix training size when 'split_type'='time'

### DISCUSSION THREAD

**Discussion by ffernest** [microsoft/FLAML#1253] [Q&A]
Hi everyone,

I noticed that FLAML will incorporate the training data with the latest test set for the next fold during the CV of time-splitted data. Is there any setting I can fix the training size, i.e. adjust the training set start time when incorporate the new test set? It is the same behaviour of the max_train_size attribute in TimeSeriesSplit of sklearn. Thanks for any suggestions.

**Comment by sonichi** [comment]
You can pass a customized splitter to [`split_type`](https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML#data-split-method)

**Reply by ffernest** [reply]
I am just wondering if it can be customized. I will try it out. Thanks!

**Reply by ffernest** [reply]
I can directly assign the TimeSeriesSplit object to "split_type" and it behaves exactly as I want. Thanks for your help!
